src/Strategy/NuGet/Nuspec.hs

```diff
@@ -0,0 +1,109 @@
+module Strategy.NuGet.Nuspec
+  ( discover
+  , strategy
+  , buildGraph
+  , analyze
+  , parseNuspec
+
+  ) where
+
+import Prologue
+
+import qualified Data.Map.Strict as M
+import qualified Data.Text as T
+import qualified Data.List as L
+import           Polysemy
+import           Polysemy.Error
+import           Polysemy.Output
+import qualified Text.XML.Light as XML
+
+import           Diagnostics
+import           DepTypes
+import           Discovery.Walk
+import           Effect.ReadFS
+import           Graphing (Graphing, unfold)
+import           Types
+
+discover :: Discover
+discover = Discover
+  { discoverName = "nuspec"
+  , discoverFunc = discover'
+  }
+
+discover' :: Members '[Embed IO, Output ConfiguredStrategy] r => Path Abs Dir -> Sem r ()
+discover' = walk $ \_ _ files -> do
+  case find (\f -> L.isSuffixOf ".nuspec" (fileName f)) files of
+    Just file -> output (configure file)
+    Nothing -> pure ()
+
+  walkContinue
+
+strategy :: Strategy BasicFileOpts
+strategy = Strategy
+  { strategyName = "nuget-nuspec"
+  , strategyAnalyze = analyze
+  , strategyModule = parent . targetFile
+  , strategyOptimal = NotOptimal
+  , strategyComplete = NotComplete
+  }
+
+analyze :: Members '[ReadFS, Error ReadFSErr] r => BasicFileOpts -> Sem r (Graphing Dependency)
+analyze BasicFileOpts{..} = do
+      contents <- readContentsBS targetFile
+      let nuspecGroups = parseNuspec =<< XML.parseXMLDoc contents
```

Comment:
can we extract this pattern into helpers in `ReadFS`? -- similar to json/parser/yaml